### PR TITLE
Arc string titles tlc

### PR DESCRIPTION
**Pull request by Volvo-hackangel**
primary tlc needs another

### CODE REVIEW

File: system_strings/sysmes_text.en [hunk 1]
```diff
@@ -630,33 +630,33 @@ ciel
...
+Back at the mansion.
+I tell her the truth.
+I lie and try to fool her.
+Heading to the park at night.
+Den. 
+At the park.
+Searching for the backdoor
+Encounter
+...Stay cautious but alert.
+Sleeping on the street
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:658]: Might strengthen this to 
> Passing out in the street

or something based on the ctx

File: system_strings/sysmes_text.en [hunk 2]
```diff
@@ -672,23 +672,23 @@ ciel
...
+...Noel-sensei.
+Getting up
+Maid's Devotion
+Have Prof. Arach take a look
+I'd much rather Kohaku-san do it
+It's now the afternoon.
+Walk around the first floor
+Walk around the second floor
+...Investigate Makihisa's study.
+...Not pursue this any further.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:687]: Doesn't work as well without the choice prompt completing the sentence, maybe
> I shouldn't pursue this further.

File: system_strings/sysmes_text.en [hunk 3]
```diff
@@ -672,23 +672,23 @@ ciel
...
+Death.
+Awakening in my room.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:689]: Good candidate to convert to active voice.
> I awake in my room.

File: system_strings/sysmes_text.en [hunk 4]
```diff
@@ -588,28 +588,28 @@ To Vlov's Stronghold
...
+No way, I'm not an adult.
+My house is a battlefield.
+Tohno Shiki has fled.
+Pursuit and Misunderstanding
+Vampire Murder Case
+Like a Sparkling Dream 
+The City of Lines
+Talk with Arcueid a little more
+More details regarding vampires
+What are the 27 Ancestors?
```
[the remark this answers is not in the document]
Comment (Volvo-hackangel) [reply system_strings/sysmes_text.en:607]: should i remove the 27 part?

[the remark this answers is not in the document]
Comment (Volvo-hackangel) [reply system_strings/sysmes_text.en:607]: it's currently "...What are the Twenty-Seven Ancestors?" in the tl should we change that?

File: system_strings/sysmes_text.en [hunk 5]
```diff
@@ -630,33 +630,33 @@ ciel
...
+Do my utmost to focus on lesson
+Do my utmost to ignore her
+Let's talk to Arihiko.
+Go to the abandoned hospital?
+I should go home and prepare.
+Back at the mansion.
+I tell her the truth.
+I lie and try to fool her.
+Heading to the park at night.
+Den.
```
[the remark this answers is not in the document]
Comment (Volvo-hackangel) [reply system_strings/sysmes_text.en:653]: i like lair better

**Pull request merged**